### DISCUSSION TITLE
Option to select menu position?

### DISCUSSION THREAD

**Discussion by ignacionelson** [GeorgianStan/vanilla-context-menu#35] [Ideas & Suggestions]
Hi! 
This is a really useful tool and has already saved me time and headaches.
I was wondering if there's a setting to set the position of the menu.
It might be a colliding dep in my case, but it always starts at the bottom right of the pointer and since I'm using it on a navbar at the top, it gets partially hidden.
Thanks!

**Comment by GeorgianStan** [comment]
Hi @ignacionelson ! Can you provide a screenshot or a short video/GIF with your use case?

**Comment by ignacionelson** [comment]
Hope this helps! I'll try messing with the parents css to see if that makes a difference

https://giphy.com/gifs/meFrNPWaaNq2okL8el

**Reply by GeorgianStan** [reply]
Thank you @ignacionelson! I understand, I don't think that modifying your CSS will solve the problem as it's the expected behavior. When the library was develop, the base use case was that a context menu would be used on an area that is larger (width x height) than the context menu itself. Your scenario, is a valid one 😊, however in the current state, the library will try to fit the context-menu inside your button and that's why it starts from bottom right. 

In the following version will update this.  Thank you for your feedback.

**Reply by ignacionelson** [reply]
That would be great, thanks!
